src/DotvvmAcademy.Web/Course/010_concepts/en/30_controls.md

```diff
@@ -8,18 +8,19 @@ CodeTask:
 
 # Controls
 
-__Controls__ are DotVVM's reusable components. You access them with the `dot` prefix.
+DotVVM also add the concept of __Controls__, reusable components that can be added anywhere in the HTML. You can recognize them easily by the `<dot:` prefix.
 
 ```dothtml
 <dot:TextBox Text="{value: Difference}" />
 ```
-The `TextBox` control is essentially an `<input>` element that allows you to use bindings.
 
-> Note: Technically, every standard HTML element is also a control, specifically an _HtmlGenericControl_.
+The `TextBox` control is essentially an `<input type="text">` element that supports data-bindings.
```

Comment:
Data-bindings are in Knockout. I think we should use Bindings to generalize Value Bindings, Commands, etc.